source/parameters/bend.md

```diff
@@ -0,0 +1,193 @@
+%---------------------------------------------------------------------------------------------------
+(s:bend.params)=
+## BendP: Bend Parameters
+
+The `BendParams` stores the parameters that characterize the shape of a [`Bend`](#s:bend) element. 
+The only relavent shape parameter that is not in the `BendParams` is the
+length `L` which is in the `LengthParams`.
+
+The parameters of this group are:
+```{code} yaml
+bend_type         - Type of bend. Default: SECTOR.
+angle             - Reference bend angle.
+g                 - Reference bend strength = 1/radius.
+bend_field_ref    - Reference bend field.
+L_chord           - Chord length.
+tilt_ref          - Reference tilt.
+e1                - Entrance end pole face rotation.
+e2                - Exit end pole face rotation.
+e1_rect           - Entrance end pole face rotation.
+e2_rect           - Exit end pole face rotation.
+edge_int1         - Entrance end fringe field integral.
+edge_int2         - Exit end fringe field integral
+```
+
+```{figure} figures/bend.svg
+:name: f:bend
+:width: 90%
+ 
+Bend geometry. Red dots are the entry and exit points that define the origin for the
+coordinate systems at the entry end {math}`(s_1, x_1)` and exit ends {math}`(s_2, x_2)` respectively.
+
+A) Bend geometry with positive bend angle. For the geometry shown,
+`g`, `angle`, `rho`, `e1`, `e2`, `e1_rect`, and `e2_rect` are all positive.
+
+B) Bend geometry with negative bend angle. For the geometry shown,
+`g`, `angle`, `rho`, `e1`, `e2`, `e1_rect`, and `e2_rect` are all negative.
+Note: The figures are drawn for zero `ref_tilt` where the rotation axis is parallel to the
+{math}`y`-axis.
+```
+
+In detail:
+- **angle**
+The total Reference bend angle. A positive `angle` represents a
+bend towards negative {math}`x` as shown in {numref}`f:bend`.
+%
+- **bend_field_ref**
+The `bend_field_ref` parameter is the reference magnetic bending field which is the field
+that is needed for the reference particle to be bent in a circle of radius `rho`
+and the placement of lattice elements downstream from the bend. The actual ("total") field is
+a vector sum of
+`bend_field_ref` plus the value of the `Bn0`  and `Bs0` multipoles. If `tilt0` and `Bs0`
+are zero, the actual field is
+  ```{code} yaml
+  B-field(total) = bend_field_ref + Bn0
+  ```
+  See the discussion of `g` and `Kn0` below for more details.
+%
+- **bend_field (output param), norm_bend_field (output_param)**
```

Comment:
Are output parameters discussed elsewhere in the standard?  This is really part of the analysis of the lattice--is it necessary or desirable to support these at the standard level?  If so, I would move them together into a separate section at the end, rather than including them in the definition of the parameter group.

Reply:
Yes the discussion was expanded in the section `User Settable (Input) and Dependent (Output) parameters.` And it is important to have these. More work is need on documenting which parameters are input and which are output but that is for a later PR.